ukrainian/knife/despair/despair.md

```diff
@@ -0,0 +1,5 @@
+З'ївши ніж, ти розумієш, що нічого не залишається, як:
+
+[Взяти мобілочку та перевірити новини у фуйсбуці](../facebook/facebook.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.